lib/email.js

```diff
@@ -0,0 +1,42 @@
+import config from '../config';
+import nodemailer from 'nodemailer';
+
+const baseUrl = 'http://' + config.app.host + ':' + config.app.clientport + '/';
+
+var transporter = nodemailer.createTransport({
+  service: 'Mandrill',
+  auth: {
+    user: config.mandrill.user,
+    pass: config.mandrill.password
+  }
+});
+
+export function sendResetPasswordEmail (uniqueIdentifier, email) {
+  const mailOptions = {
+    to: email,
+    from: 'andrew@redcometlabs.com',
```

Comment:
Could you change this from a hardcoded email to a defined email address in the config.